Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/OraFileIntentTest.java

```diff
@@ -0,0 +1,152 @@
+		Intent intent = new Intent();
+		Uri receivedUri = createTestImageFile();
+		Bitmap receivedBitmap = null;
+
+		try {
+			receivedBitmap = FileIO.getBitmapFromUri(resolver, receivedUri, launchActivityRule.getActivity().getBaseContext());
+		} catch (Exception e) {
+			Log.e("Can't Read", "Can't get Bitmap from File");
+		}
+
+		Objects.requireNonNull(receivedBitmap);
+		intent.setAction(Intent.ACTION_EDIT);
+		intent.setData(receivedUri);
+		intent.setType("image/*");
+		intent.putExtra(Intent.EXTRA_STREAM, receivedUri);
+
+		launchActivityRule.launchActivity(intent);
+		Intent mainActivityIntent = launchActivityRule.getActivity().getIntent();
+
+		String intentAction = intent.getAction();
+		String intentType = intent.getType();
+		Bundle intentBundle = intent.getExtras();
+		Objects.requireNonNull(intentBundle);
+		Uri intentUri = (Uri) intentBundle.get(Intent.EXTRA_STREAM);
+
+		String mainActivityIntentAction = mainActivityIntent.getAction();
+		String mainActivityIntentType = mainActivityIntent.getType();
+		Bundle mainActivityIntentBundle = mainActivityIntent.getExtras();
+		Objects.requireNonNull(mainActivityIntentBundle);
+		Uri mainActivityIntentUri = (Uri) mainActivityIntentBundle.get(Intent.EXTRA_STREAM);
+		Bitmap mainActivityIntentBitmap = null;
+		Objects.requireNonNull(mainActivityIntentUri);
+
+		try {
+			mainActivityIntentBitmap = FileIO.getBitmapFromUri(resolver, mainActivityIntentUri, launchActivityRule.getActivity().getBaseContext());
+		} catch (Exception e) {
+			Log.e("Can't read", "Can't get Bitmap From File");
+		}
+
+		Objects.requireNonNull(mainActivityIntentBitmap);
+
+		assertEquals(intentAction, mainActivityIntentAction);
+		assertEquals(intentType, mainActivityIntentType);
+		assertEquals(intentUri, mainActivityIntentUri);
+		assertEquals(receivedBitmap.getWidth(), mainActivityIntentBitmap.getWidth());
+		assertEquals(receivedBitmap.getHeight(), mainActivityIntentBitmap.getHeight());
+	}
+
+	private Uri createTestImageFile() {
+		Bitmap bitmap = Bitmap.createBitmap(400, 400, Bitmap.Config.ARGB_8888);
+
+		ContentValues contentValues = new ContentValues();
+		contentValues.put(MediaStore.Images.Media.DISPLAY_NAME, "testfile.ora");
+		contentValues.put(MediaStore.Images.Media.MIME_TYPE, "image/*");
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
+			contentValues.put(MediaStore.Images.Media.RELATIVE_PATH, Environment.DIRECTORY_PICTURES);
+		}
+
+		Uri imageUri = resolver.insert(MediaStore.Images.Media.EXTERNAL_CONTENT_URI, contentValues);
+		try {
+			OutputStream fos = resolver.openOutputStream(Objects.requireNonNull(imageUri));
+			assertTrue(bitmap.compress(Bitmap.CompressFormat.PNG, 100, fos));
+			assert fos != null;
+			fos.close();
+		} catch (IOException e) {
+			throw new AssertionError("Picture file could not be created.", e);
+		}
+
+		File imageFile = new File(imageUri.getPath(), "testfile.ora");
+		deletionFileList.add(imageFile);
+		return imageUri;
+	}
+}
```

Comment:
Could you please add a newline at the end of the file

Reply:
done.